class NotFoundException(Exception):
    pass

class ValidationError(Exception):
    pass

